src/telliot_feeds/cli/utils.py

```diff
@@ -42,7 +69,7 @@ def build_feed_from_input() -> Optional[DataFeed[Any]]:
     """
```

Comment:
Add a bit more to the docstring for how this is used. Either that or link to the docs where you'll cover usage